plugin/trino-hudi/src/main/java/io/trino/plugin/hudi/HudiConfig.java

```diff
@@ -191,4 +194,17 @@ public HudiConfig setMaxOutstandingSplits(int maxOutstandingSplits)
         this.maxOutstandingSplits = maxOutstandingSplits;
         return this;
     }
+
+    @NotNull
+    public String getUserInputInstantTimeFormat()
+    {
+        return userInputInstantTimeFormat;
+    }
+
+    @Config("hudi.user-input-instant-time-format")
+    @ConfigDescription("Supported user input instantTime format for the incremental query.")
+    public void setUserInputInstantTimeFormat(String userInputInstantTimeFormat)
+    {
+        this.userInputInstantTimeFormat = userInputInstantTimeFormat;
+    }
```

Comment:
Please update  `hudi.rst` and `TestHudiConfig`

Reply:
removed